sdk/face/azure-ai-vision-face/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0b1 (1970-01-01)
```

Comment:
please remember update date `1970-01-01` to real date before merge.

Reply:
Updated at commit: 0309e20e50e376e577972c0ca56e0a645a7ec42a.

Reply:
I marked it as `Unreleased` for now.